calendar-cli.py

```diff
@@ -139,8 +143,14 @@ def _calendar_addics(caldav_conn, ics, uid, args):
         raise ValueError("Nothing to do/invalid option combination for 'calendar add'-mode; either both --icalendar and --nocaldav should be set, or none of them")
         return
 
-    c = find_calendar(caldav_conn, args)
-    c.add_event(ics)
+    try:
+        c = find_calendar(caldav_conn, args)
+        c.add_event(ics)
+    except:
+        if "200 OK" in str(e):
+            print("Duplicate")
+        else:
+            raise e
```

Comment:
"raise e" and just "raise" makes a difference in the traceback.  Probably it's better with simply "raise" here rather than "raise e".